src/Components/Votes/Votes.jsx

```diff
@@ -1,5 +1,31 @@
 import React from "react";
+import { useState } from "react";
+import { updateVotesAdd, updateVotesDec } from "../../utils/API";
+import { Button } from "./Votes.styles";
 
-export default function Votes() {
-  return <div>Votes</div>;
-}
+const Votes = ({ votes, article_id }) => {
+  const [vote, setVote] = useState(0);
+
+  const voteUp = () => {
+    setVote((currentVotes) => currentVotes + 1);
+    updateVotesAdd(article_id).catch();
```

Comment:
conditional rendering needs to be undone if we it the catch block, at the moment we have an uncaught error if the user is offline